src/include/bootloader/routines/dialogs.rb

```diff
@@ -128,7 +128,7 @@ def MainDialog
 
       )
       if ret != :back && ret != :abort && ret != :cancel
-        @return_tab = CWMTab.LastTab
+        @return_tab = CWMTab.LastTab || "tab"
```

Comment:
Yes, it is a bit tricky. In grub2 due to different design it do not exists, so it is replaced by installation tab. So I use it here as default value.